test/Transforms/memref-dataflow-opt.mlir

```diff
@@ -247,3 +247,35 @@ func @store_load_store_nested_fwd(%N : index) -> f32 {
 // CHECK-NEXT:  %{{.*}} = affine.load %{{.*}}[%{{.*}}] : memref<10xf32>
 // CHECK-NEXT:  return %{{.*}} : f32
 }
+
+// CHECK-LABEL: func @should_not_fwd
+func @should_not_fwd(%A: memref<100xf32>, %M : index, %N : index) -> f32 {
+  %cf = constant 0.0 : f32
+  affine.store %cf, %A[%M] : memref<100xf32>
+  // CHECK: affine.load %{{.*}}[%{{.*}}]
+  %v = affine.load %A[%N] : memref<100xf32>
+  return %v : f32
+}
+
+// Can store forward to A[%j, %i], but no forwarding to load on %A[%i, %j]
+func @refs_not_known_to_be_equal(%A : memref<100 x 100 x f32>, %M : index) {
```

Comment:
Missing a CHECK-LABEL here.

Reply:
Thanks, fixed.

Reply:
I can't find any information on the failed check below.